### DISCUSSION TITLE
Issue not posting

### DISCUSSION THREAD

**Discussion by JoeWinograd** [mmikeww/AHK-v2-script-converter#293] [General]
Hi Folks,
I tried to submit an Issue, but it does not seem to have posted. I thought that maybe it was because I had a slash in the title (Conversion Error with InputBox followed by MsgBox in Switch/Case statements), so I changed it to a hyphen (Conversion Error with InputBox followed by MsgBox in Switch-Case statements) and tried resubmitting, but it still has not posted, as far as I can tell. Anyone know what's going on? Obviously, I'm a GitHub newbie. :) Regards, Joe

**Comment by Banaanae** [comment]
Silly question, but did you press `Submit new issue` at the bottom of the page?
If you did you should be able to post it here, then I can transfer to an issue

**Reply by JoeWinograd** [reply]
Hi Banaanae,

> did you press Submit new issue at the bottom of the page?

Yes, twice. Maybe I'm in a "not allowed to post an Issue" state because I never posted an issue before or because I'm not a Collaborator?

> If you did you should be able to post it here, then I can transfer to an issue

I'll try again at Issues. If it fails again there, I'll post it here in Discussions so you can move it. Thanks, Joe

**Reply by JoeWinograd** [reply]
Third time was a charm. :) Can't explain it, unless posting in Discussions first "activated" me to be allowed to post in Issues.
